CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/locking/ValueStore.java

```diff
@@ -67,41 +67,41 @@ public void setObject(final Object value) {
     }
 
     public final void save() {
-        converter.save();
+        converter.save(access);
     }
 
-    private static interface Saver extends Serializable {
-
-        public void save();
+    private interface Saver extends Serializable {
+        
```

Comment:
A minor comment but when possible please review your NetBeans 11 settings and consider changing it to trim trailing whitespaces.

![image](https://user-images.githubusercontent.com/39325530/78152098-17621e00-7485-11ea-8d8a-eaa42ade197d.png)